.github/workflows/rust.yaml

```diff
@@ -0,0 +1,41 @@
+name: Rust CI
+on:
+  pull_request:
+    types: [ 'opened', 'edited', 'reopened', 'synchronize' ]
+  workflow_call:
```

Comment:
Is this required?

Reply:
This seems to categorise events on the PR. New: opened, edited: new commit. Its defined in other repo also.

Reply:
No, in fact work_flow call is not needed